seacrowd/sea_datasets/xstorycloze/xstorycloze.py

```diff
@@ -0,0 +1,177 @@
+import csv
+import itertools
```

Comment:
remove this as it's unused (from code formatted)
```suggestion
import itertools
```